docs/application/dotnet/guides/app-management/event.md

```diff
@@ -0,0 +1,291 @@
+    | `SystemEvents.OutgoingMsg.EventName`          | `SystemEvents.OutgoingMsg.IdKey`                  | `msg_id`: Message ID of the sent message (string of the unsigned `int` type value) | -                                         |           -                               |
+    
+- alarm-manager
+
+  - time
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.TimeChanged.EventName`          | N/A                     | N/A                                      | When  the system time setting has changed. | You  can get the current value with the `Applications.AlarmManager.GetCurrentTime` method. |
+
+- setting
+
+  - time
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.TimeZone.EventName`             | `SystemEvents.TimeZone.Key`               | The  value of this key is the time zone value of the time zone database, for  example, "Asia/Seoul", "America/New_York". For more  information, see the IANA Time Zone Database. | When  the time zone has changed.         | You  can get the current value with the `System.SystemSettings.LocaleTimeZone` property. |
+
+  - locale
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.HourFormat.EventName`           | `SystemEvents.HourFormat.Key`             | - `SystemEvents.HourFormat.Value12` <br>- `SystemEvents.HourFormat.Value24`                             | When  the `hour_format` has changed,  for example, by the user toggling the date and time settings for the 24-hour  clock (where **OFF** stands  for the 12-hour clock). | You  can get the current value with the `System.SystemSettings.LocaleTimeFormat24HourEnabled` property. |
+    | `SystemEvents.LanguageSet.EventName`          | `SystemEvents.LanguageSet.Key`           | The value of  this key is the full name of the locale, for example, `ko_KR.UTF8` for Korean and `en_US.UTF8` for American English. For more information, see the Linux  locale information. | When the `language_set` has changed.       | You can get the  current value with the `System.SystemSettings.LocaleLanguage` property. |
+    | `SystemEvents.RegionFormat.EventName`         | `SystemEvents.RegionFormat.Key`           | The  value of this key is the full name of the locale, for example, `ko_KR.UTF8` for the Korean region  format and `en_US.UTF8`  for the USA region format. For more information, see the Linux locale  information. | When  the `region_format` has changed.     | You  can get the current value with the `System.SystemSettings.LocaleCountry` property. |
```

Comment:
@divyk I have a question. It looks like the meaning of  "Event data Value" of `SystemEvents.LanguageSet.EventName` and `SystemEvents.RegionFormat.EventName` are same, but in different sentence. Is it ok?

Reply:
@mijin85cho , I feel there is a slight difference in the meaning, because 'SystemEvents.LanguageSet.EventName' is used for language set and 'SystemEvents.RegionFormat.EventName' is used for the region. As far as my understanding the difference is visible and needed.
This is based on my limited knowledge, please correct me if I am wrong.

Reply:
Hello, @kilig. Would you please check this comment?

Reply:
Divyk is right. They are a little different.

Reply:
Thanks for the confirmation.